lang/chicken-scheme/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2019 Jerônimo Cordoni Pellegrini <j_p@aleph0.info>
+#
+# This file is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for details
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=chicken-scheme
+PKG_VERSION=5.0.0
+PKG_RELEASE:=1
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/chicken-$(PKG_VERSION)
+PKG_SOURCE:=chicken-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://code.call-cc.org/releases/5.0.0/
+PKG_MD5SUM:=de22ded475766fceaec3f9db2238ad76
+
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/chicken-scheme
+SECTION:=lang
+CATEGORY:=Languages
+TITLE:=Chicken Scheme
+URL:=https://call-cc.org
+MAINTAINER:=Jeronimo Pellegrini <j_p@aleph0.info>
+endef
+
+MAKE_FLAGS += PLATFORM=linux
+
+define Package/chicken-scheme/description
+  Chicken is an implementation of the Scheme language. It is portable, efficient, and supports 
+  the R5RS and R7RS (work in progress) standards, and many extensions.
+  Chicken can be used as a scripting language to automate tasks.
+  This package contains the interpreter, 'csi', only --
+  the compiler and the package installer are not included because they depend on a C compiler.
+endef
+
+define Build/Compile
+	$(MAKE) -C $(PKG_BUILD_DIR) PLATFORM=linux C_COMPILER=$(TARGET_CC) LINKER=$(TARGET_CC) $(TARGET_CONFIGURE_OPTS) PREFIX=/usr
```

Comment:
Does it also accept `TARGET_CFLAGS` and other flags to allow for customizations from the base build system.  For example, optimize for size `-Os` , or other hardening flags in include/hardening.mk.

Reply:
You can also just provide MAKE_FLAGS="blah blah" instead of completely overwriting the compile rule, which will play better with parallel jobs.  You can look in the repo for other users of this.